### PR TITLE
My solution to Advanced level - task 'is isogram'

### PR DESCRIPTION
**Pull request by DianaKyrychuk**

### CODE REVIEW

File: src/isIsogram.test.js [hunk 1]
```diff
@@ -8,12 +8,22 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
+    expect(typeof isIsogram('word')).toBe('boolean');
+  });
 
+  it(`empty strig is an isogram`, () => {
```
Comment (Oleksandra-Pasenchenko) [inline src/isIsogram.test.js:14]:
```suggestion
  it(`empty string is an isogram`, () => {
```

File: src/isIsogram.test.js [hunk 2]
```diff
@@ -8,12 +8,22 @@ describe(`Function 'isIsogram':`, () => {
...
+    expect(isIsogram(' ')).toBe(true);
```
Comment (Oleksandra-Pasenchenko) [inline src/isIsogram.test.js:15]: Empty string is `''`, without spaces

File: src/isIsogram.test.js [hunk 3]
```diff
@@ -8,12 +8,22 @@ describe(`Function 'isIsogram':`, () => {
...
   });
 
-  it(`should be case sensitive`, () => {
+  it(`should NOT be case sensitive`, () => {
+    expect(isIsogram('Man')).toBe(true);
+  });
```
Comment (Oleksandra-Pasenchenko) [inline src/isIsogram.test.js:20]: You don't check case sensitivity in this test, you should check what happens if the same letter is uppercased and lowercase, like `'Bob'`

File: src/isIsogram.test.js (hunk 2 above)
Comment (DianaKyrychuk) [reply src/isIsogram.test.js:15]: oh, thanks!

**Pull request closed without merging**